README.md

```diff
@@ -231,6 +245,31 @@ xhr.XMLHttpRequest = MockXMLHttpRequest
 xhr.XDomainRequest = MockXDomainRequest
 ```
 
+## Query string support
+There are many ways to stringify query parameters; consequently, `xhr` makes no
+assumptions about how to handle them, and does not support the `qs` option out
+of the box.
+
+To support the `qs` option, you must define an `xhr.queryStringStringify`
```

Comment:
pls also remove `you must`, it reads better as:

"To support a `qs` option, define an `xhr...` function"